configs/a64fx.yaml

```diff
@@ -29,10 +29,15 @@ Queue-Sizes:
   Load: 40
   Store: 24
 Branch-Predictor:
```

Comment:
Some TX2 diagrams note it's use of a multi-history branch predictor. I assume this is TAGE-like so maybe apply this config update to the TX2 YAML as well?

Reply:
Yes, that sounds like it would be.  I've updated the TX2 config as well.